src/plugins/mer/mergeneraloptionswidget.ui

```diff
@@ -137,6 +137,16 @@
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QCheckBox" name="importQmakeVariablesCheckBox">
+        <property name="toolTip">
+         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Applies to import qmake variables from headless Sailfish OS build engine virtual machine to project qmake step.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+        </property>
+        <property name="text">
+         <string>Import qmake variables from build engine</string>
```

Comment:
As I said it will be hard to say something meaningful here :) The current label and tooltip doesn't really describe what it actually does (and what are the benefits and drawbacks). The feature is not about build engine; it is about qmake invocation according to the recipe in the .spec file.

Reply:
I try one more time, then I'll ask you for assistance.:)